query/QueryManager.ts

```diff
@@ -39,13 +39,13 @@ export class QueryManager {
     public match(query: string, options?: GraknOptions): Stream<ConceptMap> {
         const matchQuery = new Query.Req().setMatchReq(
             new Graql.Match.Req().setQuery(query));
-        return this.iterateQuery(matchQuery, options ? options : new GraknOptions(), (res: Transaction.Res) => res.getQueryRes().getMatchRes().getAnswerList());
+        return this.iterateQuery(matchQuery, options ? options : new GraknOptions(), (res: Transaction.Res) => res.getQueryRes().getMatchRes().getAnswerList().map((map) => ConceptMap.of(map)));
```

Comment:
I believe this can be simplified to:
```suggestion
        return this.iterateQuery(matchQuery, options ? options : new GraknOptions(), (res: Transaction.Res) => res.getQueryRes().getMatchRes().getAnswerList().map(ConceptMap.of));
```